frontend/kesaseteli/employer/src/__tests__/utils/backend/backend-nocks.ts

```diff
@@ -0,0 +1,59 @@
+import faker from 'faker';
+import BackendEndpoint from 'kesaseteli/employer/backend-api/backend-endpoints';
+import getBackendDomain from 'kesaseteli/employer/backend-api/get-backend-domain';
+import Company from 'kesaseteli/employer/types/company';
+import nock from 'nock';
+import User from 'shared/types/user';
+
+const authenticatedUser: User = {
+  national_id_num: '111111-111C',
+  given_name: faker.name.findName(),
+  family_name: faker.name.findName(),
+  name: faker.name.findName(),
+};
+
+const expectedCompany: Company = {
+  id: 'id',
+  name: 'Acme Oy',
+  business_id: '0877830-0',
+  industry: 'Taloustavaroiden vähittäiskauppa',
+  street_address: 'Vasaratie 4 A 3',
+  postcode: '65350',
+  city: 'Vaasa',
+};
+
+export const expectAuthorizedReply = (persistValue = false): User => {
```

Comment:
as mentioned another comment, these nocks perhaps should be replaced with something else that works in parallel tests. One idea is to mock the axios library.